cmd/apps/openfaas_app.go

```diff
@@ -148,6 +157,36 @@ func MakeInstallOpenFaaS() *cobra.Command {
 		if err != nil {
 			return err
 		}
+		autoscaler, err := command.Flags().GetBool("autoscaler")
+		if err != nil {
+			return err
+		}
+		dashboard, err := command.Flags().GetBool("dashboard")
+		if err != nil {
+			return err
+		}
+
+		if dashboard {
+			// Private key
+			priv, _ := ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
```

Comment:
Can you extract these into their own privately scoped methods?

Reply:
I moved this to a separate function.